CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+**Bug fixes**:
+
+- Trim some fields in replays to the maximum expected length. ([#3706](https://github.com/getsentry/relay/pull/3706))
```

Comment:
It applies to all fields that have a `max_size`, right?

```suggestion
- Trim fields in replays to the maximum expected length. ([#3706](https://github.com/getsentry/relay/pull/3706))
```

Reply:
> It applies to all fields that have a max_size, right?

Yes, but not all fields have a max_size. I'm open to change it but not sure if it's better.

Reply:
`some` sounds arbitrary to me, that's why I would remove it. Maybe 
```suggestion
- Trim fields in replays to their defined maximum length. ([#3706](https://github.com/getsentry/relay/pull/3706))
```